doc/run_with_hive.md

```diff
@@ -0,0 +1,51 @@
+# Run SQLFlow with Hive via HiveServer2
+
+This document is a tutorial on how to run SQLFlow, which connects to the hive server2.
+
+For the most production environment, the system administrators may setup hive server with [authentication configuration](https://cwiki.apache.org/confluence/display/Hive/Setting+Up+HiveServer2#SettingUpHiveServer2-Authentication/SecurityConfiguration): e.g., KERBEROS, LDAP, PAM, or CUSTOM.
+
+## Connect Hive Server with No SASL
+
+Launch your standalone hive server Docker container by running:
```

Comment:
Maybe add some explanation that this is for test only, user can connect to a production hive server according to their own setup.

Reply:
done.